store/src/components/featured-products/FeaturedProductCard.js

```diff
@@ -11,9 +11,9 @@ const FeaturedProductCard = ({ product }) => {
           <p className="card-text p-2 d-flex justify-content-center">
```

Comment:
Change p-2 to py-2. p-2 causes the description to be off-center on small screens.